windows/src/global/delphi/packages/Keyman.System.PackageInfoRefreshKeyboards.pas

```diff
@@ -230,7 +230,7 @@ function TPackageInfoRefreshKeyboards.IsKeyboardFileByName(f: TPackageContentFil
 
   // Need to test if the JS is a valid keyboard file.
   // This is a bit of a pain... but we have to stick with .js for compat
-  if IsKeyboardFileByContent(f) then
+  if not FileExists(f.FileName) or IsKeyboardFileByContent(f) then
```

Comment:
This addresses a crash in Keyman Developer when loading a package source file where the compiled keyboards are not currently available.